lib/fetch/sanitize.js

```diff
@@ -125,6 +125,17 @@ export function sanitizeTicket(json) {
   };
 }
 
+const categories = {
+  Leadership: 'leadership',
```

Comment:
@asavin We need a hard-coded list because:
* We need to turn categories into slugs
* We need to stop invalid categories appearing as filters on the website